### DISCUSSION TITLE
Code sequence for SetUp

### DISCUSSION THREAD

**Discussion by saroj-17** [Orchid-DOTNET/BscCSIT-2023#19] [Announcements]
Steps using Visual Studio Code: 

**1. Create one dotnet folder and then open terminal 
2. git clone  https://github.com/Orchid-DOTNET/BscCSIT-2023.git
3. Get inside BscCSIT-2023 folder (cd BscCSIT-2023)
4.  git checkout -b  SarojBhandari (Here enter your name in same pattern)
5.  cd “Section A” 
6.  mkdir	 SarojBhandari (Your Name folder inside Section) 
7. dotnet new console –framework net6.0 (version 7.0 for this use dotnet –version and check your version). Here rename program.cs file to Assignment.cs file 
8. cd  ..
9. cd  .. (do this until you are back to the root folder BscCSIT-2023) 
10.  git  add .
11.  git  commit -m “Assignment added”
12.  git  push origin SarojBhandari (Your Name) 
13.  git  pull origin EmmaThapaFaculty 
14. Finally you are done.** 
![Screenshot from 2023-03-24 22-48-22](https://user-images.githubusercontent.com/115420182/227611599-da422a37-69cc-4458-bd87-1e9ef3ca68ea.png)

**Comment by enigma037** [comment]
@sarojiam17 This is good Saroj. Thanks.
Can you also write steps on how to pull request after all the work done?
Pull request is for reviewing purpose.

**Comment by saroj-17** [comment]
![image](https://user-images.githubusercontent.com/115420182/227768109-5541dd5e-8571-41e2-b500-0caa7e912e18.png)

**Reply by saroj-17** [reply]
This is for pull request after all the work are done
